packages/types/src/parser-options.ts

```diff
@@ -1,4 +1,5 @@
 import { Lib } from './lib';
+import { Program } from 'typescript';
```

Comment:
lets ensure that this isn't ever pulled in as a runtime dep on this package.

```suggestion
import type { Program } from 'typescript';
```